File: src/j4g/kotlinmaiden/command/Blame.java
package j4g.kotlinmaiden.command;

import j4g.kotlinmaiden.component.Command;
import j4g.kotlinmaiden.component.Help;
import j4g.kotlinmaiden.component.Name;

@Name("blame")
@Help("blame <name> - Blames against someone")
public class Blame implements Command {

    @Override
    public boolean test(String[] args) {
        return args.length > 1;
    }

    @Override
    public void execute(String[] args) {
        System.out.println("&$(@%# " + args[1]);
    }
}

File: src/j4g/kotlinmaiden/command/Blame.kt
package j4g.kotlinmaiden.command

import j4g.kotlinmaiden.component.*
import j4g.kotlinmaiden.component.Help

@Name("blame")
@Help("blame <name> - Blames against someone")
class Blame : Command {
    override fun test(args: Array<String>): Boolean {
        return args.size > 1
    }

    override fun execute(args: Array<String>) {
        println("&$(@%# " + args[1])
    }
}